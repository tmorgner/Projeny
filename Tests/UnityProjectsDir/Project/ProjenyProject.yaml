AssetsFolder:
- GunShip
- TextMesh Pro
- AddressableAssetsData
ProjectSettingsPath: '[ProjectRoot]/ProjectSettings'
PluginsFolder:
- SteamVR
- UnityTools
- UnityTestTools
- UnityCompileFixes
- Force Field
- Realistic Explosions Pack
- Unique Projectiles Volume 1
- UnityAssetUsageDetector
- FluentAssertions
- NaughtyAttributes
- UnityStandardAssets-Environment
- ClassicSkybox
- WeaponSystem
TargetPlatforms:
- Windows
- UWP
