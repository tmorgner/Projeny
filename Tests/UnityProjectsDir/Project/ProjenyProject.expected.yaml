AssetsFolder:
- GunShip
- TextMesh Pro
- AddressableAssetsData
PluginsFolder:
- SteamVR
- UnityTools
- UnityTestTools
- UnityCompileFixes
- Force Field
- Realistic Explosions Pack
- Unique Projectiles Volume 1
- UnityAssetUsageDetector
- FluentAssertions
- NaughtyAttributes
- UnityStandardAssets-Environment
- ClassicSkybox
- WeaponSystem
ProjectSettingsPath: '[ProjectRoot]/ProjectSettings'
Targets:
- Target: Windows
- Target: UWP
